services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,52 @@
+# TODO: Should these be separate files?
+---
+apiVersion: v1
```

Comment:
in the long term, we should build a helm chart to wrap all these files. there’s currently no way for customers to configure apps that don’t use HelmReleases. 

I know this is necessarily customer facing but I’m thinking things like increasing cpu/memory, etc.


it might also make it easier to test without needing to clone this repo or install kommander to deploy it.

Reply:
> in the long term, we should build a helm chart to wrap all these files. there’s currently no way for customers to configure apps that don’t use HelmReleases.

@mikolajb argued that we should try to avoid a `HelmRelease` if possible.

Reply:
I see, we can discuss further but using a helm chart has many pros like: long-term maintainability, testing, and configuration. it definitely require more work to get it setup with releases, etc but imo its worth it to make our lives easier in the future.

Reply:
Yes, we can defiantly reevaluate this assumption once we have it running. It will probably be slightly faster to get MVP if we don't have the chart 🤔

Reply:
sounds good, we can sync up later on 👍